src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -121,19 +123,19 @@ static Map<String, String> getHeaders(String apiKey) {
         return headers;
     }
 
-    private static javax.net.ssl.HttpsURLConnection createEasyPostConnection(final String url, final String apiKey)
-            throws IOException {
-        URL easypostURL = null;
+    private static javax.net.ssl.HttpsURLConnection createEasyPostConnection(final String url, final String apiKey,
+                                                                             final String method) throws IOException {
```

Comment:
Is this a checkstyle thing? Seems like a super odd indentation?

Reply:
Looks fine on my end, think it was a GitHub diff thing.